hibernate-core/src/main/java/org/hibernate/spi/NavigablePath.java

```diff
@@ -87,14 +83,12 @@ public NavigablePath(
 			String localName,
 			@Nullable String alias,
 			String identifierForTableGroup,
-			FullPathCalculator fullPathCalculator,
```

Comment:
Please add a deprecated constructor that still accepts the `FullPathCalculator`.

Reply:
Done, I doubt anyone was using this apart from our own `TreatedNavigablePath` though.